frontend/src/data/utils/datasetutils.ts

```diff
@@ -146,6 +146,7 @@ export const getLowestN = (
   listSize: number
 ): Row[] => {
   return data
+    .filter((row: Row) => !isNaN(row[fieldName]) && row[fieldName] !== null)
```

Comment:
Is it possible the data has undefined in it? You can use `!=` instead of `!==` to capture both null and undefined

Reply:
It is possible, did not know that you could do that, done!